include/boost/math/fft/real_algorithms.hpp

```diff
@@ -184,13 +192,13 @@
           // j=0;
           T* u = out + i, *v = out + i + prev_len;
           T Bu = *u, Bv = *v;
-          *u = (Bu + Bv)*0.5;
-          *v = (Bu - Bv)*0.5;
+          *u = Bu + Bv;
+          *v = Bu - Bv;
```

Comment:
previously, the combined effect of all the 0.5 factors amounted for the final 1/n.